src/ui/components/modal/ExportDataModal.tsx

```diff
@@ -27,20 +28,22 @@ export const ExportDataModal: FC<ExportDataModalProps> = ({
   onClose,
   variant = 'export',
 }) => {
+  const formatter = useFormatter();
```

Comment:
Changed this component to show the date range for which the report is being generated, added translations and passing `from`/`to` variables to the actions